client/src/components/Workflow/WorkflowDropdown.vue

```diff
@@ -205,6 +212,23 @@ export default {
                     });
             }
         },
+        onRestore: function () {
+            const id = this.workflow.id;
+            const name = this.workflow.name;
+            const confirmationMessage = this.l(`Are you sure you want to restore workflow '${name}'?`);
+            if (window.confirm(confirmationMessage)) {
```

Comment:
I don't think I'd bother with a confirmation here -- it's not a destructible action or anything that can't be reverted in two clicks.

Reply:
In fact we could drop the confirmation for deleting, since now that is not a destructive action also.

Reply:
@martenson Good call, +1

Reply:
done!

Reply:
I am getting confirmation window for *deleting*. Do we still want to drop it?

Reply:
I think the suggestion from Dannon and Martin was to drop it for deleting, since it's now possible to easily undelete.